marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/smart-mastering/matching/custom-xqy-matching-algo.xqy

```diff
@@ -0,0 +1,22 @@
+xquery version "1.0-ml";
```

Comment:
Can this go under smart-mastering/algorithms/custom instead? I think that gives you a nice clean slate to work with too, instead of modifying the plumbing for all the existing tests in the matching directory.

Reply:
I think of /algorithms as having tests that are limited to algorithm implementations. These changes test how matching calls algorithms, so I don't think it belongs under algorithms. The plumbing changes match what other mastering suites are doing (loading artifacts and custom modules).